elements/rh-card/demo/inline-promo.html

```diff
@@ -0,0 +1,243 @@
+<rh-card class="inline-promo">
```

Comment:
```suggestion
<rh-card variant="inline-promo">
```

Reply:
i think this should be a variant

Reply:
to elaborate: we don't want to make the value of `class` a public api on our elements

Reply:
Resolved in [089d9ac](https://github.com/RedHat-UX/red-hat-design-system/pull/1569/commits/089d9ac0b3de92225ae4d65cfba046e54340d7a2).